src/styles/spacing/index.md.njk

```diff
@@ -193,6 +193,18 @@ For example:
 - `govuk-!-padding-right-5` will apply 15px of padding to the right side of the element on small screens, increasing to 25px on large screens
 - `govuk-!-margin-0` will remove all margins at all screen sizes
 
+### Static spacing override classes
+
+The static spacing override classes start with `govuk-!-static` followed by either `margin-` or `padding-`, and then the spacing unit you want to apply.
+
+To apply spacing in a single direction, include `left-`, `right-`, `top-`, or `bottom-` between `margin-` or `padding-` and the spacing unit.
+
+For example:
+
+- `govuk-!-static-margin-9` will apply a 60px margin to all sides of the element at all screen sizes
+- `govuk-!-static-right-5` will apply 25px of padding to the right side of the element at all screen sizes
+- `govuk-!-static-margin-0` will remove all margins at all screen sizes
```

Comment:
Note that there's no difference between `govuk-!-static-margin-0` and `govuk-!-margin-0`, so we have two ways of doing the same thing.

We could exclude this example, or potentially update https://github.com/alphagov/govuk-frontend/pull/2672 so that we don't have a static override class for 0 and users should just use the responsive version instead?

(The same is true for spacing units 1-3 which are all the same size on mobile and desktop, but 0 feels ‘special’ somehow? :thinking:)